packages/formstr-app/src/formLifecycle.test.ts

```diff
@@ -0,0 +1,139 @@
+import { Event, SimplePool, generateSecretKey, getPublicKey } from 'nostr-tools';
+import { bytesToHex } from '@noble/hashes/utils';
+import { createForm } from './nostr/createForm';
+import { sendResponses } from './nostr/common';
+import { fetchFormResponses } from './nostr/responses';
+import { Response, Tag } from './nostr/types';
+
+const TEST_RELAYS = ['wss://relay.damus.io/', 'wss://nos.lol'];
```

Comment:
can we setup a smalll simple relay just for testing? something like https://github.com/inaridiy/simple-nostr-relay

so that we dont end up spamming public relays?